Make MinPath use different glpk library than it comes with.
Author: Filip Kružík (INUITS)
--- MinPath-1.6_fix-glpsol-path.py	2022-01-11 19:05:58.000000000 +0100
+++ MinPath.py	2022-01-11 20:43:56.981520000 +0100
@@ -20,13 +20,14 @@
 import math
 
 minpath = os.path.dirname(os.path.abspath(sys.argv[0]))
+glpkroot = os.getenv("EBROOTGLPK")
 print("minpath directory: " + minpath)
 if not os.path.exists(minpath + "/data"):
 	minpath = os.environ.get('MinPath')
 if not (os.path.exists(minpath) and os.path.exists(minpath + "/data")):
         sys.exit("Environment variable MinPath not set")
 
-keggPath0, seedPath0, mapPath0, glpsol0 = minpath + "/data", minpath + "/data", minpath + "/data", minpath + "/glpk-4.6/examples/glpsol"
+keggPath0, seedPath0, mapPath0, glpsol0 = minpath + "/data", minpath + "/data", minpath + "/data", glpkroot + "/bin/glpsol"
 
 def intmatrix(dim1, dim2):
 	mat = []
